core/controller/src/main/scala/whisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -86,6 +126,21 @@ class InvokerPool(
 
         case GetStatus => sender() ! invokerStatus.toMap
 
+        case msg: InvocationFinishedMessage => {
+            // Forward message to invoker, if InvokerActor exists
+            invokers.get(msg.name).map(_.forward(msg))
+        }
+
+        case msg: ActivationRequest => {
```

Comment:
I talked to Markus, because he has worked a bit more with actors.
He agreed, that there could be an impact, but he is not sure.

My idea was to handle the complete communication with the Invokers in `InvokerSupervision`, to implement in the Load Balancer only the algorithm for the dispatching of the actions to their invokerInstances.